src/script.js

```diff
@@ -40,17 +15,67 @@ createTheModal = () => {
       globalModal.style.display = "none";
     }
   }
+  console.log('window', window);
 }
 
-let modalInformation = document.getElementById("modalInformation")
-console.log('modalInformation', modalInformation);
+globalInfo = () => {
 
+  modal.style.display = "block";
+  //Create Basic information about school
+  const createDiv = document.createElement('div');
+  const info = document.getElementById("modalInformation");
+  const paragraph = document.createElement("p");
+  createDiv.appendChild(paragraph);
+  const globalInfo = document.createTextNode('Starting March 29, 2020');
+  info.append(globalInfo);
+  //Link to final Project
+  const linkToFinalProjectDiv = document.createElement('div')
+  const linkInfo = document.getElementById("linkToProject");
+  const linkToProject = document.createElement("a");
+  linkToProject.setAttribute('href', "https://github.com/JLMartinez1980");
+  linkToProject.innerText = 'final Project';
+  linkToFinalProjectDiv.appendChild(linkToProject);
+  linkInfo.append(linkToProject);
+  console.log("createDiv", createDiv);
+}
 
 const globalButton = document.getElementById("globalButton");
```

Comment:
Naming here is also a bit suspect. If I came to this after you, I would have _no idea_ what "globalButton" is